src/sonic-ctrmgrd/ctrmgr/container_startup.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/python3
+
+import argparse
+import datetime
+import inspect
+import subprocess
+import syslog
+import time
+import json
+
+from swsscommon import swsscommon
+
+# DB field names
+SET_OWNER = "set_owner"
+
+CURRENT_OWNER = "current_owner"
+UPD_TIMESTAMP = "update_time"
+DOCKER_ID = "container_id"
+REMOTE_STATE = "remote_state"
+VERSION = "container_version"
+SYSTEM_STATE = "system_state"
```

Comment:
what is the difference between the system_state here v.s feature enable/disable state?

Reply:
btw, I cannot find the definition in the design document for this system state. [https://github.com/Azure/SONiC/blob/b0b6080712d0a27c0a4748e921972901a8e3ab65/doc/kubernetes/Kubernetes-support.md](https://github.com/Azure/SONiC/blob/b0b6080712d0a27c0a4748e921972901a8e3ab65/doc/kubernetes/Kubernetes-support.md) am i missing something here?

---
In reply to: [545212259](https://github.com/Azure/sonic-buildimage/pull/5421#discussion_r545212259) [](ancestors = 545212259)

Reply:
system_state == status per systemctl for the feature. 
The state field in "show feature status" only reflect the set config, not the current status. In normal circumstances, the state configured == systemctl state. But in a small window, where the feature is crashing, configured state could be "enabled" but systemctl state could be down. But the system_state would precisely reflect the current status per "systemctl".

Based on our final resolve, let me update the doc.

Reply:
Doc updated.